### PR TITLE
Add the search feature

### PR DESCRIPTION
**Pull request by max611**
### What issue does this pull request address?
The `d3-flame-graph` allows searching a string through the graph. This can be useful when trying to see how many times you load the same snippet or section.

### What is the solution
Use the `search` feature.

### What should the reviewer focus on and are there any special considerations?
Is there a better way to do this?

### Screenshot
image

### CODE REVIEW

File: src/devtools.ts [hunk 1]
```diff
@@ -25,18 +29,56 @@ chrome.devtools.inspectedWindow.eval(
...
+    });
+}
+
+function search() {
+  const searchParam = (document.querySelector(
+    selectors.searchParam,
+  ) as HTMLInputElement).value;
+
+  if (typeof liquidFlamegraph.flamegraph !== 'undefined') {
+    liquidFlamegraph.flamegraph.search(escape(searchParam));
```
Comment (max611) [inline src/devtools.ts]: Do I need to escape since we don't show this anywhere?

Comment (wizardlyhel) [reply src/devtools.ts]: I would escape it since we are searching through the flamegraph nodes

Comment (wizardlyhel) [reply src/devtools.ts]: Where is `liquidFlamegraph.flamegraph.search` defined? is this a undocumented functionality from the Flame graph lib?

Comment (max611) [reply src/devtools.ts]: I also did not find it in their doc. I was able to find here through their [example](http://martinspier.io/d3-flame-graph/).

The source code for this is [here](https://github.com/spiermar/d3-flame-graph/blob/gh-pages/index.html).

Comment (wizardlyhel) [reply src/devtools.ts]: lol okay

**Comment by wizardlyhel** [comment]
One little thing: Please update changelog as well

**Pull request merged**